From private repo

v-anpasi opened the pull request.

azurecla commented:
Hi __@jpoon__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

deneha commented:
@jpoon Thanks for the content suggestion!  I'll go ahead and make sure  this link gets fixed properly.

azurecla commented:
Hi __@jpoon__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

anumjs committed:
Changed name to Wingtip Tickets SaaS Database Per Tenant

PRMerger19 committed:
Merge pull request #27691 from kgremban/edge-core

Adding docker step for IoT Edge on Core

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into npm2

Commit:
screenshot resolution

divyaswarnkar committed:
update5

divyaswarnkar committed:
update6

anumjs committed:
Updated image names

danieldobalian committed:
Changed AAD Dev link to docs and edited text

anumjs committed:
Fixed typos

kgremban committed:
Merge branch 'release-iot-edge-v2' of https://github.com/MicrosoftDocs/azure-docs-pr into edge-pullbbsteps

kgremban committed:
take out windows and linux articles

kgremban committed:
take out windows and linux articles

kgremban committed:
remove changes from v1

tfosmark committed:
Merge pull request #27674 from msebolt/asa-update

final revision per Chrissie Chi

barclayn committed:
removing references to the classic portal and adding some images and removing 1

barclayn committed:
adding more information to help with the decision making process of creating an Azure server on-premises or in the cloud

PRMerger15 committed:
Merge pull request #27676 from kgremban/edge-pullbbsteps

IoT Edge tutorial cleanup 1

kgremban committed:
Merge conflict

Commit:
Merge pull request #27534 from kgremban/mvc-tutorial2

Tutorial next steps

kgremban committed:
Merge branch 'release-iot-edge-v2' of https://github.com/MicrosoftDocs/azure-docs-pr into edge-pullbbsteps2

barclayn committed:
fixing formatting issue with one of the images

PRMerger15 committed:
Merge pull request #27679 from kgremban/edge-pullbbsteps2

IoT Edge tutorial cleanup 2

anumjs committed:
changed data warehouse to analytics store

JYOTHIRMAISURI committed:
minor updates to ITSMC articles

JYOTHIRMAISURI committed:
minor updates

JYOTHIRMAISURI committed:
minor updates

JYOTHIRMAISURI committed:
updates

JYOTHIRMAISURI committed:
minor update

JYOTHIRMAISURI committed:
border to image

JYOTHIRMAISURI committed:
image border fix

JYOTHIRMAISURI committed:
minor update

rayne-wiselman committed:
update a number of files and TOC entries

rayne-wiselman committed:
Update toc.yml

Commit:
freshness89

rothja committed:
Scoping limits topic with Dev Guide TOC

zjalexander committed:
Clarify scope of Update Management

Remove language around "Azure Virtual machines" since we manage all sorts of machines.

zjalexander committed:
Fix typo

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into npm2

cherylmc committed:
Update vpn-gateway-p2s-rootcert-include.md

alkohli committed:
Add note that VMware tools installation results in unsupported config

HeidiSteen committed:
edits

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cortana-analytics-playbook-vehicle-telemetry

alkohli committed:
Added note that 6.5 is not supported.

steelanddata committed:
Fixing formatting regression

steelanddata committed:
Merge bFFSranch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into working-branch-2017-11-14

Commit:
fix numbering

alkohli committed:
Update storsimple-ova-system-requirements.md

v-anpasi committed:
Resolved merge conflict per Mimi

Commit:
remove index.md and new yml LP in MVC format

paulathurman committed:
edit pass: cortana-analytics-playbook-vehicle-telemetry articles

Commit:
update screenshot

kgremban committed:
IoT Edge csharp tutorial

kgremban committed:
Iot Edge csharp tutorial 2

ajlam committed:
Update Get Started & Provision for DB per tenant

kgremban committed:
IoT Edge function tutorial prereqs

kmouss committed:
Remove  blocked functinality

Remove  blocked functinality

ajlam committed:
Add new multi-tenant db tutorials

ajlam committed:
revert db per tenant commit

kgremban committed:
IoT Edge ASA broken links

MicrosoftGuyJFlo committed:
Update ms.reviewer to reflect new PM for MFA content

ajlam committed:
revert revert

Commit:
Updated VM considerations article

mmacy committed:
[acr] update to sync with new sample

Commit:
update link to point to yml to remove pub warning

paulathurman committed:
edit pass: cortana-analytics-playbook-vehicle-telemetry article

ajlam committed:
Update welcome page

v-shils committed:
Merge pull request #27621 from mimig1/11132017DocumentDB

Cosmos DB: Change DocumentDB account name to SQL (document)

ajlam committed:
fix typos from acrolinx

tfosmark committed:
Merge pull request #26679 from kmouss/master

Add support of Azure Hybrid conversion

Commit:
removing numbering

Commit:
Merge pull request #27765 from kgremban/edge-csharp

IoT Edge csharp tutorial

PRMerger18 committed:
Merge pull request #27770 from kgremban/edge-function

IoT Edge function tutorial prereqs

sethmanheim committed:
Freshness and updating all links to new mref format

kgremban committed:
merge conflict

rmca14 committed:
Merge pull request #27723 from JYOTHIRMAISURI/master

minor updates to ITSMC articles

v-shils committed:
Merge pull request #27746 from rothja/patch-24

Scoping limits topic with Dev Guide TOC

rayne-wiselman committed:
Update concepts-on-premises-to-azure-networking.md

rayne-wiselman committed:
Update concepts-on-premises-to-on-premises-networking.md

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cluster-tutorial

v-shils committed:
Merge pull request #27758 from cherylmc/patch-1

Update vpn-gateway-p2s-rootcert-include.md

sethmanheim committed:
Acrolinx

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gw

ajlam committed:
Update "ad-hoc analytics" for acrolinx

v-shils committed:
Merge pull request #27748 from zjalexander/patch-1

Clarify scope of Update Management

kgremban committed:
Added public container images for tempsensor

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge pull request #27766 from kgremban/edge-csharp2

Iot Edge csharp tutorial 2

PRMerger19 committed:
Merge pull request #27147 from mmccrory/patch-2

Update cloud-services-configuration-and-management-faq.md

Commit:
updated release notes 

release notes updated for 1.14.0 and 1.13.0 releases

kgremban committed:
Merge branch 'release-iot-edge-v2' of https://github.com/MicrosoftDocs/azure-docs-pr into edge-imageuri

rwike77 committed:
updated parameters table

markgalioto committed:
tweaking number of RS vaults per subscription in Backup limits

GitHubber17 committed:
Merge pull request #27761 from HeidiSteen/heidist-master

Navigation to the filtering section

Commit:
updated release notes

updated release notes for version 2.3.0

rwike77 committed:
added links

v-shils committed:
Merge pull request #27688 from cherylmc/npm2

OpenSSL

Lisaco88 committed:
Merge pull request #27681 from divyaswarnkar/update

Update

itechedit committed:
Update sap-ascs-ha-multi-sid-wsfc-shared-disk.md

rmca14 committed:
Merge pull request #27764 from hning86/master

Add update screenshot in known issues article.

Commit:
Merge pull request #27781 from kgremban/edge-links

IoT Edge ASA broken links

PRMerger18 committed:
Merge pull request #27782 from kgremban/edge-imageuri

Added public container images for tempsensor

kgremban committed:
Clean up broken links

Lisaco88 committed:
Merge pull request #27762 from steelanddata/working-branch-2017-11-14

Working branch 2017 11 14

kgremban committed:
Merge branch 'release-iot-edge-v2' of https://github.com/MicrosoftDocs/azure-docs-pr into edge-imageuri

kgremban committed:
Edge agent and hub URIs

rwike77 committed:
formatting

Commit:
Updated per PM feedback

itechedit committed:
Update sap-high-availability-architecture-scenarios.md

GitHubber17 committed:
Merge pull request #27706 from danieldobalian/updating-link

Changed AAD Dev link to docs and edited text

Commit:
updated disk drive configs

ajlam committed:
Fix broken links and TOC

steveesp committed:
Azure kernel GA

steveesp committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vnet-optimal

Azure kernel GA steps

sethmanheim committed:
Typo

v-shils committed:
Merge pull request #27646 from MicrosoftGuyJFlo/MFAUpdatePM

Update ms.reviewer to reflect new PM for MFA content

ankryach committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

carolinacmoravia committed:
P: Extra space within link structure 

Hello, @linda33wj, 

Localization team have reported source content issue that cause localized version to have broken link in localized version. 

Many thanks.

rmca14 committed:
Merge pull request #26959 from HeidiSteen/heidist-indexers

Heidist indexers

ankryach committed:
Added /events WebSocket example

ankryach committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

kgremban committed:
IoT Edge UI fixes

PRMerger13 committed:
Merge pull request #27788 from kgremban/edge-links2

Clean up broken links

PRMerger12 committed:
Merge pull request #27790 from kgremban/edge-imageuri

Edge agent and hub URIs

kgremban committed:
Merge branch 'release-iot-edge-v2' of https://github.com/MicrosoftDocs/azure-docs-pr into edge-ui

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge pull request #27759 from alkohli/patch-156

Add note that VMware tools installation results in unsupported config

PRMerger16 committed:
Merge pull request #27792 from carolinacmoravia/patch-460

P: Extra space within link structure

Commit:
Merge pull request #27786 from markgalioto/ab-11142017

tweaking number of RS vaults per subscription in Backup limits

anumjs committed:
Added indentation for images in a list

tynevi committed:
Merge pull request #27763 from v-savila/1123951

new yml LP in MVC format

Commit:
added more advanced scenarios to bottom of each of these

barclayn committed:
removing user information per pr reviewer

ktoliver committed:
Merge pull request #27773 from mmacy/acr-georep-tutorial-update

 [acr] update to sync with new sample

Lisaco88 committed:
Merge pull request #27701 from MicrosoftDocs/adapted-Azure-AD-tutorial-for-SAML-SSO-for-Jira-and-confluence

Adapted Azure AD tutorial for SAML SSO for Jira and Confluence

kgremban committed:
Fewer changes in quickstart-linux

Commit:
updated with acrolinx edits

mmacy committed:
[aci] add Cloud Shell to PSh quickstart

Commit:
Merge pull request #27772 from jeffgilb/jeff1114

Updated VM considerations article

Commit:
two items addressed

JasonWHowell committed:
Adding top level node to landing page

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gw

Commit:
resolved PR issues on Clipper and recommended encoder docs

cherylmc committed:
format

gsacavdm committed:
Clarify Azure functions variations

Commit:
Merge pull request #27431 from dougeby/ebymain

Added new sections to view and delete role assignments with PowerShell

sethmanheim committed:
Freshness review and updates

kgremban committed:
More UI updates

v-shils committed:
Merge pull request #27738 from rayne-wiselman/rayne-november14

update a number of files and TOC entries

kgremban committed:
Removed changes to iot-edge-register-device

Lisaco88 committed:
Merge pull request #27157 from tamram/tamram-1107

retrieve old PS sample for calculating container size

Commit:
Merge pull request #27796 from johndeu/master

Added some next steps to the encoding pages for MES

Commit:
resolved enumeration issue on clipper getting started article

ktoliver committed:
Merge pull request #27013 from jluk/master

remove preview tags for GA and include terraform walkthrough

Commit:
removed duplicate recommended encoders list

Lisaco88 committed:
Merge pull request #27776 from sethmanheim/wk11-14

Freshness and updating all links to new mref format

ktoliver committed:
Merge pull request #27705 from anumjs/master

Updated Analytics db per tenant tutorial

PRMerger16 committed:
Merge pull request #27794 from kgremban/edge-ui

IoT Edge UI fixes

Lisaco88 committed:
Merge pull request #27793 from ankryach/master

Master

Commit:
fixed minor list issue on recommended encoders article

ShawnJackson committed:
Merge pull request #27112 from itechedit/sap-ascs-ha-multi-sid-wsfc-shared-disk

edit pass: sap-ascs-ha-multi-sid-wsfc-shared-disk

ShawnJackson committed:
Merge pull request #27270 from itechedit/sap-high-availability-architecture-scenarios

edit pass: sap-high-availability-architecture-scenarios

Lisaco88 committed:
Merge pull request #27798 from JasonWHowell/tsi_toc

TSI: Adding index.yml to the TOC.yml per Kris

Ja-Dunn committed:
Merge pull request #27715 from barclayn/identity-access

removing references to the classic portal and adding some images and …

Ja-Dunn committed:
Merge pull request #27791 from steveesp/vnet-optimal

Vnet optimal

Ja-Dunn committed:
Merge pull request #27801 from sethmanheim/sb-fresh

Freshness review and updates

MicrosoftGuyJFlo committed:
Update

MicrosoftGuyJFlo committed:
Update

GitHubber17 committed:
Merge pull request #25460 from philmea/nested-virt-patch

Nested virtualization How To

MicrosoftGuyJFlo committed:
Update

MicrosoftGuyJFlo committed:
Update

Commit:
Merge pull request #27802 from gsacavdm/patch-2

Clarify Azure functions variations

ShawnJackson committed:
Update sap-high-availability-guide-start.md

GitHubber17 committed:
Merge pull request #20382 from mmacy/aci-run-to-completion

[aci] new article: run-to-completion containers

MicrosoftGuyJFlo committed:
Update

tfitzmac committed:
redirecting policy intro

PRMerger16 committed:
Merge pull request #27785 from srinathnarayanan/patch-3

updated release notes

Commit:
Merge pull request #27783 from srinathnarayanan/patch-2

updated release notes

Ja-Dunn committed:
Merge pull request #27343 from dbgeorge/marketing

Marketing

rwike77 committed:
formatting

MicrosoftGuyJFlo committed:
Update

ShawnJackson committed:
Merge pull request #27638 from itechedit/sap-high-availability-guide-start

edit pass: sap-high-availability-guide-start

ktoliver committed:
Merge pull request #27570 from JSeb225/master

Documentation for ASA Edge

tfitzmac committed:
fixing link

Commit:
Merge pull request #27799 from cherylmc/p2sfaq

format makecert

daveba committed:
updating saas tutorial list

MicrosoftGuyJFlo committed:
Update

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gw

cherylmc committed:
graphics

rwike77 committed:
removed article

rwike77 committed:
removed article

Ja-Dunn committed:
Merge pull request #27811 from tfitzmac/1114policy

redirecting policy intro

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gw

cychua committed:
Add clarification for UPN source attribute

Add clarification for UPN source attribute - cannot be multi-valued

ShawnJackson committed:
Update cortana-analytics-playbook-vehicle-telemetry-deep-dive.md

ShawnJackson committed:
Update cortana-analytics-playbook-vehicle-telemetry-powerbi.md

v-alje committed:
Merge pull request #27216 from MicrosoftDocs/release-hdinsight-connect

Release hdinsight connect

ShawnJackson committed:
Update cortana-analytics-playbook-vehicle-telemetry.md

v-alje committed:
Merge pull request #27702 from MicrosoftDocs/release-connect-functions

Connect() Release PR for Azure Functions

v-alje committed:
Merge pull request #27823 from MicrosoftDocs/release-connect-dms

Release connect dms

cherylmc committed:
graphics

v-alje committed:
Merge pull request #27825 from MicrosoftDocs/release-databricks

Release databricks

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into gw

v-alje committed:
Merge pull request #27827 from MicrosoftDocs/release-tsi

Release tsi

v-alje committed:
Merge pull request #27822 from MicrosoftDocs/release-connect-app-insights

Release connect app insights

v-alje committed:
Merge pull request #27826 from MicrosoftDocs/release-iot-edge-v2

Release iot edge v2

v-alje committed:
Merge pull request #27824 from MicrosoftDocs/release-connect-cosmosdb

Release connect cosmosdb

rmca14 committed:
Merge pull request #27048 from MikeDodaro/video-srch-tutorial-toc

Add single-page app for video search + source code + toc changes

Sammak committed:
Fixed doc

ShawnJackson committed:
Update cortana-analytics-playbook-vehicle-telemetry-powerbi.md

ninarn committed:
Adding SaaS standalone get started article, updating TOC

rmca14 committed:
Merge pull request #27375 from rwike77/cluster-tutorial

Cluster tutorial

ShawnJackson committed:
Merge pull request #27693 from paulathurman/cortana-analytics-playbook-vehicle-telemetry

Edit pass: Cortana analytics playbook vehicle telemetry

rmca14 committed:
Merge pull request #27797 from mmacy/aci-psh-quickstart-cloudshell

[aci] add Cloud Shell to PSh quickstart

ShawnJackson committed:
Update active-directory-passwords-getting-started.md

ajlam committed:
Fix blocking issues

ninarn committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

rmca14 committed:
Merge pull request #27820 from rwike77/deleteloadtest

Delete loadtest article

ninarn committed:
Updating image

ajlam committed:
Small typo

rmca14 committed:
Merge pull request #27232 from MicrosoftGuyJFlo/SSPROOBE

Reset Password link in Login Window

ShawnJackson committed:
Update active-directory-passwords-how-it-works.md

ShawnJackson committed:
Merge pull request #27476 from LoriMerrick/active-directory-passwords-two-articles

edit pass: active-directory-passwords-two-articles

cychua committed:
Add group info

cychua committed:
Grammar update

zhongc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hft

Commit:
images for clarity, improving code sample to show response handling

ktoliver committed:
Merge pull request #27774 from ajlam/multitenantdb

Add multi-tenant DB SaaS tutorials

zhongc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hft

zhongc committed:
address feedback

zhongc committed:
address feedback

ktoliver committed:
Merge pull request #27817 from MicrosoftDocs/update-saas-tutorial-list-apex-portal

updating saas tutorial list

ktoliver committed:
Merge pull request #27819 from cherylmc/gw

Gw

ktoliver committed:
Merge pull request #27686 from Sammak/master

Adding doc for custom policy language customization

ktoliver committed:
Merge pull request #27828 from cychua/master

Add clarification for UPN source attribute

ktoliver committed:
Merge pull request #27856 from JamesEarle/emotion-api-javascript

Improving code sample, adding Images for clarity.

ktoliver committed:
Merge pull request #27061 from zhongc/hft

High frequency trading simulation on ASA first draft

ninarn committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

ninarn committed:
Adding header and updating formating

ktoliver committed:
Merge pull request #27837 from ninarn/master

Adding SaaS standalone get started article, updating TOC

Commit:
Removed documentid

ktoliver committed:
Merge pull request #27548 from Jen7714/patch-1

Create app-service-diagnostics.md

PRMerger16 committed:
Merge pull request #27745 from MarkusVi/freshness89

freshness89

v-anpasi committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
